package/rpm/magnum.spec

```diff
@@ -0,0 +1,109 @@
+Name: magnum
+URL: http://magnum.graphics
+Version: 2020.06.2972.g8b832382a
+Release: 1
+Summary: C++11/C++14 graphics middleware for games and data visualization
+License: MIT
+Source: %{name}-%{version}.tar.gz
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-buildroot
+Requires: corrade, openal-soft, mesa-libGL, mesa-libEGL, SDL2, glfw, vulkan-loader, glslang
+BuildRequires: cmake, git, gcc-c++, openal-soft-devel, mesa-libGL-devel, mesa-libEGL-devel, SDL2-devel, glfw-devel, vulkan-loader-devel, glslang-devel
```

Comment:
Just for the record, I think `glslang-devel` isn't needed here and I'm going to remove it. 

It will be needed for `GlslangShaderConverter` in the plugins tho.